Description: Disable failing component tests for gles packaging
Author: Zoltan Balogh <zoltan.balogh@canonical.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- ubuntu-ui-toolkit-1.1.1188+14.10.20140813.4.orig/tests/unit/unit.pro
+++ ubuntu-ui-toolkit-1.1.1188+14.10.20140813.4/tests/unit/unit.pro
@@ -10,8 +10,6 @@ SUBDIRS += testparser
 # https://bugs.launchpad.net/qemu-linaro/+bug/1084148
 #######################################
 #!contains(QMAKE_HOST.arch,armv7l) {
-    SUBDIRS += tst_components \
-        tst_components_benchmark


 SUBDIRS += tst_units \
--- ubuntu-ui-toolkit-1.1.1188+14.10.20140813.4.orig/tests/unit_x11/unit_x11.pro
+++ ubuntu-ui-toolkit-1.1.1188+14.10.20140813.4/tests/unit_x11/unit_x11.pro
@@ -1,7 +1,6 @@
 TEMPLATE = subdirs

-SUBDIRS += tst_components \
-    tst_ubuntu_shape \
+SUBDIRS += tst_ubuntu_shape \
     tst_page \
     tst_test \
     tst_inversemousearea \

--- ubuntu-ui-toolkit-1.1.1188+14.10.20140813.4.orig/ubuntu-sdk.pro
+++ ubuntu-ui-toolkit-1.1.1188+14.10.20140813.4/ubuntu-sdk.pro
@@ -1,40 +1,7 @@
 include( documentation/documentation.pri )

 TEMPLATE = subdirs
-SUBDIRS += modules/ubuntu-ui-toolkit.pro tests examples po/po.pro
-
-tests/unit.depends = modules/ubuntu-ui-toolkit.pro
-
-# additional 'make test' target required by continuous integration system
-test.target = test
-test.commands = make check
-test.depends = license modules/ubuntu-ui-toolkit.pro
-QMAKE_EXTRA_TARGETS += test
-
-test_performance.target = test-performance
-test_performance.commands = cd tests/unit/tst_performance; make check
-test_performance.depends = modules/ubuntu-ui-toolkit.pro
-QMAKE_EXTRA_TARGETS += test_performance
-
-test_api.target = test-api
-test_api.commands = cd tests/unit/tst_components; make check
-test_api.depends = modules/ubuntu-ui-toolkit.pro
-QMAKE_EXTRA_TARGETS += test_api
-
-test_components.target = test-components
-test_components.commands = cd tests/autopilot; python3 -m autopilot.run run ubuntuuitoolkit
-test_components.depends = modules/ubuntu-ui-toolkit.pro
-QMAKE_EXTRA_TARGETS += test_components
-
-test_components2.target = test-components2
-test_components2.commands = cd tests/autopilot; python2 -m autopilot.run run ubuntuuitoolkit
-test_components2.depends = modules/ubuntu-ui-toolkit.pro
-QMAKE_EXTRA_TARGETS += test_components2
-
-qmluitests.target = qmluitests
-qmluitests.commands = cd tests/unit_x11; make check
-qmluitests.depends = modules/ubuntu-ui-toolkit.pro
-QMAKE_EXTRA_TARGETS += qmluitests
+SUBDIRS += modules/ubuntu-ui-toolkit.pro  examples po/po.pro

 license.target = license
 license.commands = ./tests/license/checklicense.sh

